test/GameLeague.t.sol

```diff
@@ -85,7 +75,7 @@ contract GameLeagueTest is Test {
 
     function testInitializeLeague(uint256 _prizePool) public {
         _prizePool = bound(_prizePool, 10 ^ 18, 10 ^ 23);
-        vm.deal(deployer, _prizePool + 10 * 10 ^ 18);
+        vm.deal(deployer, (_prizePool + 10 * 10) ^ 18);
 
```

Comment:
This could overflow in some cases. 
Leaving it without parentheses means: prize pool + 10 eth
which can be written even better as `(prizePool + 10 ether)`